index.js

```diff
@@ -214,7 +215,16 @@ function connect(params, cb) {
 		close = true;
 		cb(new Error("Connection closes (wrong auth?)"));
 	};
-	socket = net.connect(params.port, params.host, function() {
+
+	let socketArgs = [];
+	if (params.unixSocket) {
+		socketArgs.push(params.unixSocket);
+	}
+	else {
+		socketArgs.push(params.port, params.host);
+	}
+
+	socket = net.connect(...socketArgs, function() {
```

Comment:
we support "node": ">=0.10" at the moment. Could you rewrite this line to work in node 0.10? Otherwise we have to bump the minimum node version which is a breaking change.

Reply:
I've updated the code to support the older versions of node and all tests have passed. I may look into adding support for the abstract namespace unix sockets in the future which will add support for KDB 3.5+ on Linux.